ci/playbooks/multinode-autohold.yml

```diff
@@ -3,13 +3,20 @@
   hosts: controller
   gather_facts: true
   tasks:
+    - name: Verify if "success" flag exists after successful tests execution
+      register: cifmw_success_flag
+      ansible.builtin.stat:
+        path: "{{ ansible_user_dir }}/cifmw-success"
+
     - name: Check if the current run is an authold target
```

Comment:
same typo here and also line 53 below (may as well fix if you get another update here)

Reply:
Done